doc/source/conf.py

```diff
@@ -228,7 +232,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = os.path.join(html_static_path[0], "favicon.ico")
```

Comment:
html_static_path is now not used. could remove?

Reply:
It might still be used by sphinx, not fully sure.

Reply:
The docs don't suggest this is the default value (https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_static_path), and we for sure still use this (there are a lot of files in the `_static` directory.